toJSONMap.js

```diff
@@ -0,0 +1,67 @@
+const JSONMap = {
```

Comment:
Can we move this file into the `test` directory since we are only using it to get the expected results for our unit tests?

You could even put this directly into the statetoJson.test.js file if you want